templates/cicd/pipeline_cfn.yml

```diff
@@ -234,16 +249,27 @@ Resources:
               - Name: SCCheckoutArtifact
             OutputArtifacts:
               - Name: BuildOutput
-{{$length := len .Stages}}{{if gt $length 0}}{{range $stage := .Stages}}{{$svcNum := len $stage.LocalServices}}{{if gt $svcNum 0}}
+{{- $length := len .Stages}}{{if gt $length 0}}{{range $stage := .Stages}}{{$svcNum := len $stage.LocalServices}}{{if gt $svcNum 0}}
         - Name: DeployTo-{{$stage.Name}}
           Actions:{{if $stage.Prod}}
             - Name: ApprovePromotionTo-{{$stage.Name}}
               ActionTypeId:
                 Category: Approval
                 Owner: AWS
+                Version: 1
                 Provider: Manual
+              RunOrder: 1{{end}}{{if $stage.TestCommands}}
```

Comment:
Oh shoot, i just realized that we put the stage before `CreateOrUpdate-{{$svc}}-{{$stage.Name}}` whereas it should be after it!

Can we create a separate PR moving below?

Reply:
You betcha. https://github.com/aws/amazon-ecs-cli-v2/pull/1039